tests/compose-tests/test-installroot.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+set -xeuo pipefail
+
+dn=$(cd $(dirname $0) && pwd)
+. ${dn}/libcomposetest.sh
+
+prepare_compose_test "installroot"
+compose_base_argv="--repo=${repobuild} --cache-only --cachedir=${test_compose_datadir}/cache"
```

Comment:
This was already factored out and defined in `libcomposetest.sh`, right?

Reply:
Yep, good catch.